cpp/iedriver/BrowserFactory.h

```diff
@@ -76,6 +76,7 @@ class BrowserFactory {
   static bool IsWindowsVistaOrGreater(void);
 
   bool IsEdgeMode(void) const;
+  void DeleteEdgeTempDir(void);
```

Comment:
Add declarations for `DirectoryExists()`, `CreateUniqueTempDir()`, and `DeleteDirectory()` to the `private` section of the class.

Reply:
Moved these functions to class `BrowserFactory` .